api-reference/10 UI Components/dxPivotGrid/1 Configuration/showTotalsPrior.md

```diff
@@ -6,11 +6,13 @@ default: 'none'
 ---
 ---
 ##### shortDescription
-Specifies where to show the total rows or columns. Applies only if [rowHeaderLayout](/api-reference/10%20UI%20Components/dxPivotGrid/1%20Configuration/rowHeaderLayout.md '/Documentation/ApiReference/UI_Components/dxPivotGrid/Configuration/#rowHeaderLayout') is *"standard"*.
+Specifies where to show the total rows or columns.
 
 ---
 By default, total rows and columns are shown after data (columns at the right side, rows at the bottom). You can place total rows, total columns or both before data using this property.
```

Comment:
1) `By default, total rows and columns are shown after data (columns at the right side, rows at the bottom).`

Please rewrite without using "By default".

2) `You can place total rows, total columns or both before data using this property.`

Please add a comma (an Oxford comma) after "total columns".